ROOTFS/usr/bin/waggle_sanity_check.py

```diff
@@ -143,8 +143,9 @@ def execute_tests_in_path(tests_dir, tests_severity, timeout_secs):
             totalFailed += 1
             led_set = True
 
-    # pet systemd watchdog
-    update_systemd_watchdog()
+        # pet systemd watchdog
```

Comment:
lets increase the timeout_secs value to 150 seconds to allow more time for longer running tests. we have a 180 second window, we might as well allow more use of it.